drools-wb-webapp/src/main/java/org/drools/workbench/client/home/HomeProducer.java

```diff
@@ -36,21 +36,14 @@
 /**
  * Producer method for the Home Page content
  */
-@ApplicationScoped
+@Dependent
 public class HomeProducer {
 
-    private HomeModel model;
-
-    @Inject
-    private PlaceManager placeManager;
-
-    @Inject
-    protected LibraryMonitor libraryMonitor;
```

Comment:
@ederign The `LibraryMonitorImpl` is `@ApplicationScoped` so it won't actually by created just by injecting it (it would have to actually be used). And the `PlaceManager` is definitely used (indirectly) by several entry points, so I don't think there is any side-effect from deleting these unused references.